### ISSUE TITLE
[Term Entry] Nmap: TCP Connect Scan

### ISSUE THREAD

**Issue by SSwiniarski** [Codecademy/docs#1315]
### Reason/inspiration (optional)

We would like an entry for [Nmap ](https://www.codecademy.com/resources/docs/cybersecurity/nmap) about the TCP Connect Scan option.

It should be located under docs/content/cybersecurity/concepts/nmap/terms/tcp-connect-scan/tcp-connect-scan.md

It should include:
- A description of what the option does.
- A listing of the option itself: `-sT`
- A syntax section showing the option in use.
- An example showing Nmap's output with the option. (Use `scanme.nmap.org` as a target.)

You can look at [Aggressive Scan](https://www.codecademy.com/resources/docs/cybersecurity/nmap/aggressive-scan) to see an example of what we'd like to see.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

**Comment by joesizzlin** [comment]
I would like to claim this issue. Thank you

**Comment by SSwiniarski** [comment]
@joesizzlin, You've been assigned!

**Comment by joesizzlin** [comment]
@SSwiniarski, I was wondering if it would be okay to include a picture (from a Codecademy skillpath lesson) to better describe/present the role 3-way handshake has with TCP Connect Scan?
https://static-assets.codecademy.com/skillpaths/learn-nmap/Performing_Basic_Network_Scans/network_scan_01.png

**Comment by yangc95** [comment]
@joesizzlin feel free to add images! For Docs, add the images to the media/ folder and link to it like: https://raw.githubusercontent.com/Codecademy/docs/main/media/`image-name.filetype`

**Comment by SSwiniarski** [comment]
@Dusch4593, @yangc95, how do we feel about reusing assets from the rest of the Codecademy site?

**Comment by yangc95** [comment]
@SSwiniarski I think it's okay to refuse images as long as it's described in as much detail necessary for a user to understand (without being too wordy) and if the image is stored in the `media` folder (to stick with the Docs standard).
